src/OpenPlatform.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace DDB\OpenPlatform;
+
+use DDB\OpenPlatform\Request\GenericRequest;
+use DDB\OpenPlatform\Request\SearchRequest;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class OpenPlatform
+{
+    /**
+     * HTTP client.
+     *
+     * @var \Symfony\Contracts\HttpClient\HttpClientInterface
+     */
+    protected $client;
+
+    /**
+     * OpenPlatform access token.
+     *
+     * @var string
+     */
+    protected $token;
+
+    public function __construct($token, HttpClientInterface $client = null)
+    {
+        $this->token = $token;
+
+        if (!$client) {
+            $client = HttpClient::create();
+        }
+        $this->client = $client;
+    }
+
+    public function searchRequest()
```

Comment:
```suggestion
    public function searchRequest() : searchRequest
```

If we are going with no/light DocBlocks I would like to see a return type definition. This goes for other methods as well.